eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -17,6 +14,14 @@ parameters:
     type: string
     default: ''
     displayName: 'Path to API specification file (TypeSpec/OpenAPI)'
+  - name: ApiVersion
+    type: string
+    default: ''
```

Comment:
Given you're checking for empty on this below, do you want to also remove `default` here to make it required?

Reply:
It's required param in the entry yml file.

Reply:
I forgot that the default must be set to support PR trigger scenario. However, even if I leave default value as empty string, it would still display the parameter in pipeline UI as required.